src/main/java/frc/robot/constants/BlankConstants.java

```diff
@@ -1,9 +1,14 @@
 package frc.robot.constants;
 
-import frc.robot.drive.MotorType;
+import frc.robot.motors.MotorType;
 
 public class BlankConstants implements Constants {
```

Comment:
This is a good example where a class comment would help - why do we need a blank constants class?